src/main/java/com/epam/ta/reportportal/entity/widget/content/CumulativeTrendChartEntry.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.ta.reportportal.entity.widget.content;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class CumulativeTrendChartEntry {
```

Comment:
add documentation that says that it is not a database entry

Reply:
added